data/tutorials/language/4ad_01_operators.md

```diff
@@ -93,21 +93,22 @@ OCaml has a subtle syntax; not everything is allowed as an operator symbol. An o
 **Prefix Operator**
 
 1. First character
-  * `?` `~` or
-  * `!`
+    * `?` `~` or
+    * `!`
 1. Following characters, at least one if the first character is `!`, optional otherwise
-  * `$` `&` `*` `+` `-` `/` `=` `>` `@` `^` `|` or
-  * `%` `<`
+    * `$` `&` `*` `+` `-` `/` `=` `>` `@` `^` `|` or
+    * `%` `<`
 
 **Binary Operator**
+
 1. First character
-  * `$` `&` `*` `+` `-` `/` `=` `>` `@` `^` `|` or
-  * `%` `<` or
-  * `#`
-2. Following characters, at least one if the first character is `#`, optional otherwise
-  * `$` `&` `*` `+` `-` `/` `=` `>` `@` `^` `|` or
-  * `%` `<` or
-  * `!` `.` `:` `?` `~`
+    * `$` `&` `*` `+` `-` `/` `=` `>` `@` `^` `|` or
+    * `%` `<` or
+    * `#`
+1. Following characters, at least one if the first character is `#`, optional otherwise
+    * `$` `&` `*` `+` `-` `/` `=` `>` `@` `^` `|` or
+    * `%` `<` or
```

Comment:
```suggestion
    * `$` `&` `*` `+` `-` `/` `=` `>` `@` `^` `|`
    * `%` `<`
    * `#`
1. Following characters, at least one if the first character is `#`, optional otherwise
    * `$` `&` `*` `+` `-` `/` `=` `>` `@` `^` `|`
    * `%` `<`
```